src/config/texts.js

```diff
@@ -1531,7 +1531,7 @@ export const texts = {
     hintCityAndStreet: 'Bitte geben Sie Ihre Ortschaft und anschließend Ihre Straße an.',
     hintStreet: 'Bitte geben Sie Ihre Straße an.',
     location: 'Ortschaft',
-    myLocation: 'Meine Straße:',
+    myLocation: 'Meine Straße',
```

Comment:
whats the difference here on the screenshots to look at?

![image](https://github.com/user-attachments/assets/68efbe7a-2c17-4a21-8cbd-a1a718857c18)

Reply:
Meine Straße has the value `:` instead of `::`